.yarnrc

```diff
@@ -0,0 +1 @@
+registry "https://registry.yarnpkg.com"
```

Comment:
As this is a public package, when adding dependencies we should assure we use the public repository.